docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/bundling-a-custom-plugin-selection/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Bundling a custom plugin selection"
+date: "2023-03-22"
+sidebar_position: 600
+---
+
+The [default releases](https://github.com/snowplow/snowplow-javascript-tracker/releases) of the JavaScript Tracker include a full-featured `sp.js` version, and a smaller-filesize `sp.lite.js` version that includes less [Plugins](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/index.md).
+
+While plugins can be loaded dynamically, self hosting the additional files adds complexity and the additional resource requests can negatively impact page performance.
+For this reason, it can be desirable to have your own version of `sp.js` that includes _just_ the features you need, to both reduce the filesize and not require loading additional script resources.
+
+## Custom Plugin Selections
+To do this, you'll need to install [git](https://git-scm.com/) and [Node.js](https://nodejs.org/en/) 14 or 16 (at the time of writing) then open a Terminal or Command Prompt and run the following:
+
+```bash
+$ git clone https://github.com/snowplow/snowplow-javascript-tracker.git
+$ cd snowplow-javascript-tracker
+$ npx @microsoft/rush update
+```
+
+Then open the file `trackers/javascript-tracker/tracker.config.ts` in your favourite text editor and flip the `true` values to `false` for features which you don't require (or vice versa).
+
+Then run:
+
+```bash
+$ npx @microsoft/rush build
+```
+
+Once complete (it might take a minute or two), you'll find your brand new `sp.js` at:
+`trackers/javascript-tracker/dist/sp.js` along with a new sourcemap `sp.js.map` which we suggest hosting together for a better developer experience.
+
+## Including Custom Plugins
+The above only works for the official plugins included in the `snowplow-javascript-tracker` repository.
+If you have [developed your own plugins](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/creating-your-own-plugins/index.md), or want to include plugins from third parties, the process is similar, but with some additional small code changes:
+
+- Include the plugin as a dependency in `trackers/javascript-tracker/package.json`
```

Comment:
Do you think a numbered list would be more fitting here ?

Reply:
Yeah that makes total sense, my bad. :)